lib/vdsm/storage/managedvolume.py

```diff
@@ -258,15 +260,19 @@ def _invalidate_lvm_devices(attachment):
         lvm.invalidate_devices()
 
 
-def _silent_remove(db, vol_id):
+def _silent_remove(db, vol_id, sd_id):
```

Comment:
Lets always use sd_id, vol_id - the only place we cannot do this is the schema. The rest of the code can look like sd_id was always part of the code.